azure-pipelines.yml

```diff
@@ -124,20 +130,23 @@ jobs:
       displayName: Update MSYS2
     - script: |
         set PATH=%MSYS2_ROOT%\usr\bin;%SystemRoot%\system32;%SystemRoot%;%SystemRoot%\System32\Wbem
+        if %compiler%==gcc ( set "TOOLCHAIN=mingw-w64-$(MSYS2_ARCH)-toolchain" ) else ( set "TOOLCHAIN=mingw-w64-$(MSYS2_ARCH)-clang" )
```

Comment:
(you probably know, just in case: the toolchain group contains more than gcc, it contains the base for building a pkgbuild: https://msys2.duckdns.org/group/mingw-w64-i686-toolchain)

Reply:
I didn't know.  But I found that the only thing in that set we appear to need which isn't a dependency of clang itself is pkg-config.

I'm not sure what change you are suggesting here?

Reply:
> I'm not sure what change you are suggesting here?

No change :) Just wanted to point it out in case.